strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/storage/metadata/nuget/TempNupkgFileTest.java

```diff
@@ -45,97 +43,67 @@
 
 /**
  * @author Dmitry Sviridov
+ * @author Pablo Tirado
  */
-@ContextConfiguration(classes = { NugetBootersTestConfig.class })
+@ContextConfiguration(classes = { NugetLayoutProviderTestConfig.class })
 @SpringBootTest
 @ActiveProfiles(profiles = "test")
 public class TempNupkgFileTest
 {
-    private String baseDirectoryPath;
 
-    @Inject
-    private PropertiesBooter propertiesBooter;
-
-    @BeforeEach
-    public void setUp()
-        throws Exception
-    {
-        baseDirectoryPath = propertiesBooter.getHomeDirectory() + "/tmp/tnft";
-
-        File baseDirectory = getCleanBaseDirectory();
-        baseDirectory.mkdirs();
-    }
-
-    @AfterEach
-    public void tearDown()
-        throws IOException,
-               JAXBException
-    {
-        getCleanBaseDirectory();
-    }
-
-    private File getCleanBaseDirectory()
-        throws IOException
-    {
-        File baseDirectory = new File(baseDirectoryPath);
-
-        if (baseDirectory.exists())
-        {
-            FileUtils.deleteDirectory(baseDirectory);
-        }
-
-        return baseDirectory;
-    }
+    private static final String REPOSITORY_RELEASES = "tnft-releases";
 
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class,
+                  ArtifactManagementTestExecutionListener.class })
     @Test
-    public void testHashTempFile()
-        throws Exception
+    public void testHashTempFile(@NugetRepository(repositoryId = REPOSITORY_RELEASES)
```

Comment:
@ptirador you don't need a repository here, because we don't need to deploy artifacts within this test. The tip is that if you leave the `@NugetTestArtifact.repositoryId` empty then it wouldn't be deployed.

Reply:
Thanks for that great tip!